ops/model.py

```diff
@@ -2719,6 +2719,42 @@ def send_signal(self, sig: Union[int, str], *service_names: str):
 
         self._pebble.send_signal(sig, service_names)
 
+    def get_notice(self, id: str) -> pebble.Notice:
+        """Get details about a single notice by ID.
+
+        Raises:
+            ModelError: if a notice with the given ID is not found
+        """
+        try:
+            return self._pebble.get_notice(id)
+        except pebble.APIError as e:
+            if e.code == 404:
+                raise ModelError(f'notice {id!r} not found')
+            raise
+
+    def get_notices(
+        self,
+        *,
+        select: Optional[pebble.NoticesSelect] = None,
+        user_id: Optional[int] = None,
+        types: Optional[Iterable[pebble.NoticeType]] = None,
+        keys: Optional[Iterable[str]] = None,
+        after: Optional[datetime.datetime] = None,
+    ) -> List[pebble.Notice]:
+        """Query for notices that match the provided filters.
+
+        See :meth:`ops.pebble.Client.get_notices` for documentation of the
+        parameters.
+        """
+        notices = self._pebble.get_notices(
+            select=select,
+            user_id=user_id,
+            types=types,
+            keys=keys,
+            after=after,
+        )
+        return notices
```

Comment:
Any reason not to just do this?
```suggestion
        return self._pebble.get_notices(
            select=select,
            user_id=user_id,
            types=types,
            keys=keys,
            after=after,
        )
```

Reply:
Heh, I had it that way originally, but then I felt the `return` got lost. However, I changed it back -- this way it's clearer it's proxying straight through.